db/forms/kpi/gbv_assessment.rb

```diff
@@ -0,0 +1,28 @@
+gbv_assessment_fields = [
```

Comment:
Is the only purpose of these new forms to drive the select options for row/column on the reports editor?  If so i think we need to do this a different way... perhaps hard coding the options